meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,296 @@
+  "processingTimeMs": 0,
+  "limit": 20,
+  "offset": 0,
+  "estimatedTotalHits": 2
+}
+```
+
+
+🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/product/discussions/677).
+
+⚠️ The experimental feature API can break between Meilisearch versions. Be careful if using it in production.
+
+Done by @Kerollmops in #3825
+
+
+## Display ranking scores at search
+
+Meilisearch v1.3.0 introduces ranking scores per document via the `showRankingScore` search query parameter. 
+
+```bash 
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "Batman Returns", "showRankingScore": true }'
+```
+
+Using this search query parameter means that each document will return with a `_rankingScore` property displaying a score between 0 and 1. The higher the ranking score, the more relevant the document.
+
+```json 
+"_rankingScore": 0.8575757575757575,
+```
+
+### Ranking score details - Experimental 
+
+Additionally, as an experimental feature, you can view detailed scores per ranking rule for each document via the `showRankingScoreDetails` search query parameter. 
+
+```bash 
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "Batman Returns", "showRankingScoreDetails": true }'
+```
+
+With this search query parameter in place each document will return with an `_rankingScoreDetails` JSON object. That object will contain a property and a score value for each ranking rule. 
+
+```json 
+"_rankingScoreDetails": {
+                "words": {
+                    "order": 0,
+                    "matchingWords": 1,
+                    "maxMatchingWords": 1,
+                    "score": 1.0
+                },
+                "typo": {
+                    "order": 1,
+                    "typoCount": 0,
+                    "maxTypoCount": 1,
+                    "score": 1.0
+                },
+                "proximity": {
+                    "order": 2,
+                    "score": 1.0
+                },
+                "attribute": {
+                    "order": 3,
+                    "attributes_ranking_order": 0.8,
+                    "attributes_query_word_order": 0.6363636363636364,
+                    "score": 0.7272727272727273
+                },
+                "exactness": {
+                    "order": 4,
+                    "matchType": "noExactMatch",
+                    "score": 0.3333333333333333
+                }
+            }
+```
+
+This experimental feature can be turned on via the HTTP API using v1.3.0's new `/experimental-features` endpoint. If you are using [Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=engine&utm_medium=meilisearch) this can be enabled immeadiately after upgrade.
```

Comment:
https://github.com/meilisearch/engine-team/pull/52#discussion_r1244912339 it was in response to this review. Just wanted to highlight that it can be available on cloud moreso. Perhaps the wording confuses the message? 🤔

Reply:
I wanted to say that you can try it new experimental features on the Cloud as you no more have to reboot the bainry by hand :)

Reply:
Yes, but the users don't know this originally, I mean, they did not have the opportunity to complain they cannot use it on the cloud side. It's supposed to work on the Cloud side by default for them.
It's really confusing to me to talk about this here, and can send the opposite message of "the other features don't work with the cloud"
I would rather we highlight the cloud at the beginning of the changelog if we want to highlight it

Reply:
Ok, let's talk about the Cloud at the beginning of the changelog :)

Reply:
I've removed the references from Vector Store and Scoring and added the block we have from the readme up a tthe top instead. Thanks folks 🙌